info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -281,21 +332,53 @@ private void storeElectionContestCandidateInDatabase(JsonObject candidate,
     Entity candidateEntity =
         Entity.newBuilder(candidateKey)
             .set("name", name)
-            .set("partyAffiliation", party + " Party")
+            .set("party", party + " Party")
+            .set("email", email)
+            .set("phone", phone)
+            .set("PhotoUrl", photoUrl)
```

Comment:
Would it be more consistent for this to be `"photoUrl"` throughout instead?

Reply:
I only use that casing because that is the casing used in the API.

Reply:
In other places of InfoCompiler and the servlets (in the merged version), I used `"photoUrl"`. Sorry about that! This should be fixed when this branch gets merged.

Reply:
The API seems to use `"photoUrl"`: https://developers.google.com/civic-information/docs/v2/elections/voterInfoQuery
However, this should be fixed automatically when this PR gets merged. Perhaps chain this PR to Travis_Dev first.